### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by ishpaul777**
This is a <strong>fictional page</strong> built for my portfolio project and is built with HTML, CSS & Javascript and based on an online website for a conference Any Logos and name used is for Placeholding. Website layout and design belong to [Cindy Shin](https://www.behance.net/adagio07) and are used for educational purposes.
I have presented the project in a [Loom Video](https://www.loom.com/share/89e7cef7a9cc4eaaaf2be69a8523f9c8).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,57 @@
...
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://ishpaul777.github.io/Landing-page-for-Dev-)
 
+## Loom Video
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Loom Video Link](https://www.loom.com/share/89e7cef7a9cc4eaaaf2be69a8523f9c8)
```
Comment (tufoinnkuo10) [inline README.md:23]: - [ optional] loom video link in the pull request summary will also be of help

File: README.md [hunk 2]
```diff
@@ -1,72 +1,57 @@
...
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Ishpaul Singh**
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- [@github](https://github.com/ishpaul777)
```
Comment (tufoinnkuo10) [inline README.md:41]: - [ ] kindly add a LinkedIn handle

File: index.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+          >
+        </a>
+      </div>
+      <div class="nav-options">
+        <ul class="menu">
+          <li class="menu-item"><a href="about.html">About</a></li>
+          <li class="menu-item"><a href="#">Program</a></li>
+          <li class="menu-item"><a href="#">Join</a></li>
+          <li class="menu-item"><a href="#">Sponsor</a></li>
+          <li class="menu-item"><a href="#">News</a></li>
```
Comment (tufoinnkuo10) [inline index.html:30]: - [ ] kindly add links to the nave sections. they should take you to their respective sections

File: index.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+        </ul>
+        <a href="#" class="register-btn"> Get Tickets </a>
+      </div>
+      <!--Toggle menu screen-->
```
Comment (tufoinnkuo10) [inline index.html:34]: - [ ] great job on the mobile version. kindly put your mobile hamburger on the left like on the Behance design
your design
Screenshot (171)_LI
behance 👇🏾 
Screenshot (169)_LI

File: index.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+    <!--Navbar-->
+    <nav>
+      <div class="logo-container">
+        <a href="index.html" class="logo">
+          <span class="highlight">Dev</span>Summit<span class="highlight"
+            >.</span
+          >
+        </a>
+      </div>
+      <div class="nav-options">
```
Comment (tufoinnkuo10) [inline index.html:24]: - [ ] please you left out the top short bar section with social media links, logout, and language links aligned to the right. please fix that 
your version 👇🏾 
Screenshot (172)
behance 👇🏾 
Screenshot (173)_LI

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,913 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
```
Comment (tufoinnkuo10) [inline styles.css:5]: - [ ] kindly fix your fonts. Fonts used originally: Cocogoose, Lato

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @ishpaul777 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ code looks good
✅ mobile and desktop version looks ok

## Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md [hunk 7]
```diff
@@ -1,72 +1,56 @@
...
+- [@twitter](https://twitter.com/ishpaul777)
+- [@Linkedin](https://www.linkedin.com/in/ishpaul-singh-264590226/)
```
Comment (vechicin) [inline README.md:41]: While you did address the previous reviewer's required change, you didn't do so properly. Make sure the content that you are linking (that means the content inside the []) includes your name or your social media handle. Having these links with the social media name is not professional and it doesn't give you visibility.

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,955 @@
...
+}
+
+a {
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+}
```
Comment (vechicin) [inline styles.css:11]: You still have to address the changes to the fonts required by the previous reviewer. Please do so.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @ishpaul777 ✋, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. ⛑️ 


## Suggested changes ♻️ 
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by vechicin — APPROVED** [review]
# Hi @ishpaul777 ✋, 

In my opinion, your project is ready for presentation! 💯 🎯 
There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**